### PR TITLE
Keyboard components - add docs

### PR DESCRIPTION
**Pull request by M-i-k-e-l**
## Description
Keyboard components - add docs

## Changelog
Keyboard components - add docs

### CODE REVIEW

File: lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
+{
+  "name": "KeyboardAccessoryView",
```
Comment (lidord-wix) [inline lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.json:2]: consider adding 'api' to the file name

File: lib/components/Keyboard/KeyboardInput/keyboardRegistry.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "KeyboardRegistry",
```
Comment (lidord-wix) [inline lib/components/Keyboard/KeyboardInput/keyboardRegistry.json:2]: consider adding 'api' to the file name

File: lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.json (hunk 1 above)
Comment (M-i-k-e-l) [reply lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.json]: I did not mean to set you as reviewer, 10x :D

**Review by lidord-wix — APPROVED** [review]

File: lib/components/Keyboard/KeyboardTracking/keyboardAwareInsetsView.api.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "KeyboardAwareInsetsView",
+  "category": "keyboard",
+  "description": "Used to add an inset when a keyboard is used and might hide part of the screen.",
+  "notes": "This view is useful only for iOS.",
+  "extends": ["KeyboardTracking"],
```
Comment (Inbal-Tish) [inline lib/components/Keyboard/KeyboardTracking/keyboardAwareInsetsView.api.json:6]: extends `KeyboardTrackingView`

File: lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView/keyboardTracking.api.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "KeyboardTracking",
```
Comment (Inbal-Tish) [inline lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView/keyboardTracking.api.json:2]: name: `KeyboardTrackingView`

File: lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.api.json [hunk 5]
```diff
@@ -0,0 +1,97 @@
+{
+  "name": "KeyboardAccessoryView",
+  "category": "keyboard",
```
Comment (Inbal-Tish) [inline lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.api.json:3]: extends: `KeyboardTrackingView`

File: lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.api.json [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+    {
+      "name": "useSafeArea",
+      "type": "boolean",
+      "description": "Whether or not to handle SafeArea\niOS only.",
+      "default": "true"
+    },
+    {
+      "name": "usesBottomTabs",
+      "type": "boolean",
+      "description": "Whether or not to include bottom tab bat inset\niOS only.",
```
Comment (Inbal-Tish) [inline lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.api.json:92]:
```suggestion
      "description": "Whether or not to include bottom tab bar inset\niOS only.",
```

File: lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.api.json (hunk 5 above)
Comment (M-i-k-e-l) [reply lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.api.json:3]: It technically does, but for the user it is not true (all `KeyboardTrackingView` props are passed literally in the `KeyboardAccessoryViewProps` and there's no `extends` \ `others` used).

Comment (Inbal-Tish) [reply lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.api.json:3]: Right... I'm not sure why thought

**Review by Inbal-Tish — APPROVED** [review]

**Comment by Inbal-Tish** [comment]
@M-i-k-e-l Approved. The build fails so merge when fixed

**Pull request merged**